### ISSUE TITLE
jQuery 1.6.2 has a known vulnerability

### ISSUE THREAD

**Issue by mmucklo** [minkphp/Mink#689]
https://github.com/minkphp/Mink/search?utf8=✓&q=jquery

A scan produced this:

```
[...]/vendor/behat/mink/driver-testsuite/web-fixtures/js/jquery-1.6.2-min.js
 ↳ jquery 1.6.2-min has known vulnerabilities: severity: medium; CVE: CVE-2011-4969; http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2011-4969 http://research.insecurelabs.org/jquery/test/ severity: medium; bug: 11290, summary: Selector interpreted as HTML; http://bugs.jquery.com/ticket/11290 http://research.insecurelabs.org/jquery/test/
```

**Comment by aik099** [comment]
Sure, send a PR to use another jQuery version. This isn't critical vulnerability because:

1. Mink isn't used on production website (or it should not be used)
2. the `/vendor` folder, where Mink resides isn't accessible from the Web (or at least I hope it isn't)

**Comment by aik099** [comment]
@mmucklo , any update?

**Comment by stof** [comment]
Closes because #721 is removing the driver-testsuite from this repo entirely.
